### ISSUE TITLE
myelin oligodendrocyte glycoprotein) positive antibodies in cerebrospinal fluid

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11360]
MOG (myelin oligodendrocyte glycoprotein) positive antibodies in cerebrospinal fluid. (Presence of MOG antibodies in CSF, which mistakenly target and damage myelin)

**Your nano-attribution (ORCID)**
All of these findings can be seen in MOGAD, multiple sclerosis, and other neuroimmunological diseases.

The references supporting these labels are as follows:

 PMID: 40327177; 39318613

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001441
- label: CSF anti-myelin oligodendrocyte glycoprotein antibody positivity
- parents: [CSF autoimmune antibody positivity (HP:0032265)](https://hpo.jax.org/app/browse/term/HP:0032265)
- pmid: PMID:40327177|PMID:39318613
- def: The presence of autoantibodies (immunoglobulins) in the cerebrospinal fluid (CSF) that react against myelin oligodendrocyte glycoprotein.
- comment: CSF anti-myelin oligodendrocyte glycoprotein antibody positivity can be seen in MOG antibody-associated disease (MOGAD), multiple sclerosis, and other neuroimmunological diseases.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11360
- synonyms:
